test/test_grid_file.py

```diff
@@ -781,6 +848,7 @@ def test_zip(self):
         self.assertEqual(1, self.db.fs.chunks.count_documents({}))
 
         g = GridOut(self.db.fs, f._id)
+        g.open()
```

Comment:
Why add the open call here?

Reply:
Artifact from before I made this test sync-only, good catch.